core/src/test/scala/stryker4s/mutants/findmutants/FileCollectorTest.scala

```diff
@@ -66,9 +67,61 @@ class FileCollectorTest extends Stryker4sSuite {
         val results = sut.collectFiles()
 
         results should have size 2
+        results should contain only(basePath / "someFile.scala", basePath / "secondFile.scala")
+      }
+
+      it("Should exclude the file specified in the excluded files config") {
+        implicit val config: Config = Config(
+          files = Seq("**/someFile.scala", "**/secondFile.scala"),
+          excludedFiles = Seq("**/someFile.scala"),
+          baseDir = filledDirPath)
+
+        val sut = new FileCollector()
+
+        val results = sut.collectFiles()
+
+        results should have size 1
+        results should contain only (basePath / "secondFile.scala")
+      }
+
+      it("Should exclude all files specified in the excluded files config") {
+        implicit val config: Config = Config(
+          files = Seq("**/someFile.scala", "**/secondFile.scala"),
+          excludedFiles = Seq("**/someFile.scala", "**/secondFile.scala"),
+          baseDir = filledDirPath)
+
+        val sut = new FileCollector()
+
+        val results = sut.collectFiles()
+
+        results should have size 0
+      }
+
+      it("Should exclude all files based on a wildcard") {
+        implicit val config: Config = Config(
+          files = Seq("**/someFile.scala", "**/secondFile.scala"),
+          excludedFiles = Seq("**/*.scala"),
+          baseDir = filledDirPath)
+
+        val sut = new FileCollector()
+
+        val results = sut.collectFiles()
+
+        results should have size 0
```

Comment:
You can also use `should be (empty)`, which might be a little cleaner

Reply:
done